todolist/app/components/Input/index.js

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from 'react';
+import { TextInput } from 'react-native';
+import { connect } from 'react-redux';
+
+import { actionCreators } from '../../redux/Todo/actions';
+
+class Input extends Component {
+  state ={
+    value: null
+  };
+
+  generateId = () => {
+    return '_' + Math.random().toString(36).substr(2, 9);
```

Comment:
you can use a template literal..

Reply:
Also, extract the Math.random... part to a utils file (utils folder at the same level that redux) i.e.: numberUtils.js or anything you like